Scarb.toml

```diff
@@ -3,20 +3,20 @@ members = [
   "listings/getting-started/*",
   "listings/applications/*",
   "listings/advanced-concepts/*",
-  "listings/templates/*"
+  "listings/templates/*",
 ]
 
 [workspace.scripts]
 test = "$(git rev-parse --show-toplevel)/scripts/test_resolver.sh"
 
-# [workspace.tool.snforge]
+[workspace.tool.snforge]
 
 [workspace.dependencies]
 starknet = ">=2.6.3"
-# snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.11.0" }
-openzeppelin = { git = "https://github.com/OpenZeppelin/cairo-contracts.git", tag="v0.11.0" }
+snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.23.0" }
```

Comment:
Move forge to dev dependencies, also we will use version `0.24.0`

Reply:
moving snforge to dev dependencies throws this error 
<img width="694" alt="image" src="https://github.com/NethermindEth/StarknetByExample/assets/64266194/cbc950ed-56b8-4edd-86ab-4b0b127dcb35">